connectivity/drivers/lora/TARGET_STM32WL/STM32WL_radio_driver.h

```diff
@@ -0,0 +1,697 @@
+#define RBI_CONF_RFO_HP     2
+
+typedef enum
+{
+  RBI_SWITCH_OFF    = 0,
+  RBI_SWITCH_RX     = 1,
+  RBI_SWITCH_RFO_LP = 2,
+  RBI_SWITCH_RFO_HP = 3,
+} RBI_Switch_TypeDef;
+
+
+/**
+  * Indicates whether or not TCXO is supported by the board
+  * 0: TCXO not supported
+  * 1: TCXO supported
+  */
+#define IS_TCXO_SUPPORTED                   1U
+
+/**
+  * @brief drive value used anytime radio is NOT in TX low power mode
+  */
+#define SMPS_DRIVE_SETTING_DEFAULT  SMPS_DRV_40
+
+/**
+  * @brief drive value used anytime radio is in TX low power mode
+  *        TX low power mode is the worst case because the PA sinks from SMPS
+  *        while in high power mode, current is sunk directly from the battery
+  */
+#define SMPS_DRIVE_SETTING_MAX      SMPS_DRV_60
+
+/*!
+ * \brief Change the value on the device internal trimming capacitor
+ */
+#define REG_XTA_TRIM                                0x0911
+
+/*!
+ * \brief Change the value on the device internal trimming capacitor
+ */
+#define REG_XTB_TRIM                                0x0912
+
+/*!
+ * \brief Set the current max value in the over current protection
+ */
+#define REG_OCP                                     0x08E7
+
+/*!
+ * \brief PA Clamping threshold
+ */
+#define REG_TX_CLAMP                                0x08D8
+
+/**
+  * @brief  Sub-GHz radio register (re) definition
+  * @note   The sub-GHz radio peripheral registers can be accessed by sub-GHz radio command
+  *         SUBGRF_WriteRegisters() and SUBGRF_ReadRegisters() "
+  */
+/*Sub-GHz radio generic bit synchronization register*/
+#define SUBGHZ_GBSYNCR                              REG_BIT_SYNC
+/*Sub-GHz radio generic packet control 1A register*/
+#define SUBGHZ_GPKTCTL1AR                           REG_LR_WHITSEEDBASEADDR_MSB
+/*Sub-GHz radio generic whitening LSB register*/
+#define SUBGHZ_GWHITEINIRL                          REG_LR_WHITSEEDBASEADDR_LSB  
+/*Sub-GHz radio generic CRC initial MSB register*/
+#define SUBGHZ_GCRCINIRH                            REG_LR_CRCSEEDBASEADDR
+/*Sub-GHz radio generic CRC initial LSB register*/
+#define SUBGHZ_GCRCINIRL                            0x06BD
+/*Sub-GHz radio generic CRC polynomial MSB register*/
+#define SUBGHZ_GCRCPOLRH                            REG_LR_CRCPOLYBASEADDR
+/*Sub-GHz radio generic CRC polynomial LSB register*/
+#define SUBGHZ_GCRCPOLRL                            0x06BF
+/*Sub-GHz radio generic synchronization word control register 7*/
+#define SUBGHZ_GSYNCR7                              REG_LR_SYNCWORDBASEADDRESS
+/*Sub-GHz radio generic synchronization word control register 6*/
+#define SUBGHZ_GSYNCR6                              0x06C1
+/*Sub-GHz radio generic synchronization word control register 5*/
+#define SUBGHZ_GSYNCR5                              0x06C2 
+/*Sub-GHz radio generic synchronization word control register 4*/
+#define SUBGHZ_GSYNCR4                              0x06C3
+/*Sub-GHz radio generic synchronization word control register 3*/
```

Comment:
Why all these register definitions are not defined in CMSIS file like other registers ?